src/extension/features/general/colour-blind-mode/settings.js

```diff
@@ -5,5 +5,5 @@ module.exports = {
   section: 'general',
   title: 'Colour Blind Mode',
   description:
-    'Changes colours like red, yellow and green in the interface to colours and shapes that are more easily distinguishable by colourblind people.',
+    'Changes colours like red, yellow and green in the interface to colours and shapes that are more easily distinguishable by colourblind people. Colours and shapes are configurable in YNAB Display Options.',
```

Comment:
Here's a thought I have:

1. Default enable this feature.
2. Rename it to "Customize Color Scheme"
3. In the YNAB Display Menu, keep the color pickers but also add two options:
  a. Default Colors (selecting this would disable the color pickers)
  b. Custom (selecting this would just set the colors to our pre-defined color blind settings but also enable the color pickers)

For this to work, we'd probably want to namespace all our colors here under `body.tk-custom-colors` so that when they select "Default Colors" it actually uses those and not our own overrides which we'd add/remove when they click the buttons.

Let me know if this all makes sense